WordPress/src/main/java/org/wordpress/android/ui/posts/PostPreviewActivity.java

```diff
@@ -97,6 +98,10 @@ protected void onResume() {
         EventBus.getDefault().register(this);
         mDispatcher.register(this);
 
+        if (mPost == null) {
```

Comment:
Just noticed this uses `mPost.getId()` to get the post again. Do we know `mPost` won't be null here?